exec/java-exec/src/main/java/org/apache/drill/exec/work/filter/RuntimeFilterSink.java

```diff
@@ -115,7 +98,8 @@ public void aggregate(RuntimeFilterWritable runtimeFilterWritable) {
   public RuntimeFilterWritable fetchLatestDuplicatedAggregatedOne() {
     try {
       aggregatedRFLock.lock();
-      return aggregated.duplicate(bufferAllocator);
+      RuntimeFilterWritable duplicated = aggregated.duplicate(bufferAllocator);
```

Comment:
This class is not required anymore. It's only used in tests where we should replace the reference with correct implementation. Basically update the test implementation of [this method](https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/ops/FragmentContext.java#L172)